### PR TITLE
Implement address discovery advanced action

### PR DESCRIPTION
**Pull request by nop33**
In the PR I created a new advanced operation box in the addresses screen to create a proof-of-concept address discovery mechanism using the newly created SDK function: https://github.com/alephium/js-sdk/pull/134. Due to the heavy computational load of this function, I used a web worker to unblock React's rendering thread so that the app doesn't freeze.

Tests fail because of the usage of a local SDK package, it'll be updated once a new SDK version is released.

### CODE REVIEW

File: public/workers/addressDiscovery.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+const { ExplorerClient, walletImport, discoverActiveAddresses } = require('@alephium/sdk')
```
Comment (mvaivre) [inline public/workers/addressDiscovery.js:19]: Can't a web worker be written in typescript?

Comment (nop33) [reply public/workers/addressDiscovery.js:19]: Looks like someone managed to do it using `react-scripts v5` with a gotcha that there's a workaround to make jest work:

- https://github.com/facebook/create-react-app/issues/11651
- https://github.com/cmdcolin/cra-webpack5-web-worker-example

Comment (nop33) [reply public/workers/addressDiscovery.js:19]: Maybe we should consider using Vite on this project as well?

Comment (mvaivre) [reply public/workers/addressDiscovery.js:19]: Yes 👍 #445

**Review by mvaivre — APPROVED** [review]

**Pull request merged**